src/llmq/quorums.cpp

```diff
@@ -1003,18 +1003,24 @@ void CQuorumManager::CleanupOldQuorumData(const CBlockIndex* pIndex) const
         return;
     }
 
-    std::set<uint256> dbKeys;
+    std::set<uint256> dbKeysToSkip;
 
     LogPrint(BCLog::LLMQ, "CQuorumManager::%d -- start\n", __func__);
+    // Platform quorums in all networks are newed every 24h (LLMQ_100_67, LLMQ_25_67, LLMQ_DEVNET_PLATFORM and LLMQ_REGTEST_PLATFORM)
+    // Every month, 24 x 30 quorums are created
```

Comment:
nit: imo sounds unclear, maybe by "during a month" or something like that? Otherwise it sounds like "once a month 24*30". I'm not native so ignore me if I'm wrong